components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/java/org/wso2/carbon/identity/api/server/application/management/v1/core/functions/application/inbound/oauth2/OAuthConsumerAppToApiModel.java

```diff
@@ -99,19 +101,19 @@ private IdTokenConfiguration buildIdTokenConfiguration(OAuthConsumerAppDTO oAuth
 
         return new IdTokenConfiguration()
                 .expiryInSeconds(oAuthConsumerAppDTO.getIdTokenExpiryTime())
-                .audience(getAudiences(oAuthConsumerAppDTO))
+                .audience(getIdTokenAudiences(oAuthConsumerAppDTO.getIdTokenAudiences()))
                 .encryption(buildIdTokenEncryptionConfiguration(oAuthConsumerAppDTO));
     }
 
-    private List<String> getAudiences(OAuthConsumerAppDTO oAuthConsumerAppDTO) {
+    private List<String> getIdTokenAudiences(String[] audiences) {
 
-        if (oAuthConsumerAppDTO.getAudiences() == null) {
-            return Collections.emptyList();
-        } else {
-            return Arrays.asList(oAuthConsumerAppDTO.getAudiences());
-        }
+        return (audiences == null) ? Collections.emptyList() : Arrays.asList(audiences);
     }
 
+    private List<String> getAccessTokenAudiences(String[] audiences) {
+
+        return (audiences == null) ? Collections.emptyList() : Arrays.asList(audiences);
+    }
```

Comment:
add a new line

Reply:
Resolved with 5f1a0a83b80553100ed41fb6e6efc9f2f91dcb9f

Reply:
Resolved with 5f1a0a83b80553100ed41fb6e6efc9f2f91dcb9f

Reply:
remove the new line you have added. the new line should be added before starting the next method